exposed-json/src/test/kotlin/org/jetbrains/exposed/sql/json/JsonColumnTests.kt

```diff
@@ -79,7 +79,7 @@ class JsonColumnTests : DatabaseTestsBase() {
 
     @Test
     fun testSelectWhereWithExtract() {
-        withJsonTable(exclude = TestDB.allH2TestDB) { tester, _, data1, _ ->
+        withJsonTable(exclude = TestDB.allH2) { tester, _, data1, _ ->
```

Comment:
I would make it consistent with all other naming for constants: `ALL_H2`

Reply:
Renamed these constants, is it looks now like:
```kotlin
val ALL_H2_V1 = listOf(H2_V1, H2_V1_MYSQL)
val ALL_H2_V2 = listOf(H2_V2, H2_V2_MYSQL, H2_V2_PSQL, H2_V2_MARIADB, H2_V2_ORACLE, H2_V2_SQLSERVER)
val ALL_H2 = ALL_H2_V1 + ALL_H2_V2
val ALL_MYSQL = listOf(MYSQL_V5, MYSQL_V8)
val ALL_MARIADB = listOf(MARIADB)
val ALL_MYSQL_LIKE = ALL_MYSQL + ALL_MARIADB + listOf(H2_V2_MYSQL, H2_V2_MARIADB, H2_V1_MYSQL)
val ALL_POSTGRES = listOf(POSTGRESQL, POSTGRESQLNG)
val ALL_POSTGRES_LIKE = ALL_POSTGRES + listOf(H2_V2_PSQL)
val ALL_ORACLE_LIKE = listOf(ORACLE, H2_V2_ORACLE)
val ALL_SQLSERVER_LIKE = listOf(SQLSERVER, H2_V2_SQLSERVER)
val ALL = TestDB.entries.toList()
```